src/_data/home.json

```
@@ -0,0 +1,43 @@
+[{
+    "title": "Access and <span class=\"text-red-600\">transform any data,<\/span> over any protocol",
+    "content": "Getting access to data from your operations is key to the start of a digital transformation. Node-RED makes it easy to access and transform data over any protocol.",
```

```suggestion
    "content": "Acquiring data from your operations is key to the start of a digital transformation. Node-RED makes it easy to access and transform data over analog and digital protocols alike.",
```
Lots of "Access", so went with acquiring. `Any protocol` felt repeatitive too. What about this?